File: spring-boot/java/src/main/java/daggerok/web/IndexPage.java
package daggerok.web;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

import java.util.Locale;

@Controller
public class IndexPage {

    @GetMapping("/")
    public String index(final Locale locale,
                        final Model model) {

        model.addAttribute("locale", locale);
        return "index";
    }
}

File: spring-boot/kotlin/src/main/kotlin/daggerok/web/IndexPage.kt
package daggerok.web

import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping
import java.util.*

@Controller
class IndexPage {

  @GetMapping("/")
  fun index(locale: Locale, model: Model): String {
    model.addAttribute("locale", locale)
    return "index"
  }
}
